crates/bevy_app/src/app.rs

```diff
@@ -56,6 +56,8 @@ pub struct App {
     pub runner: Box<dyn Fn(App)>,
     /// A container of [`Stage`]s set to be run in a linear order.
     pub schedule: Schedule,
+    /// stage run after main schedule and sub app schedules
+    pub stage_very_last: SystemStage,
```

Comment:
Maybe `final`, rather than `very_last`?

Reply:
I'm down for that. `very_last` was my 🚲 🏠  name.